lib/homepage.jsx

```diff
@@ -66,19 +66,15 @@ var Values = React.createClass({
   }
 });
 
-var Homepage = React.createClass({
-  render: function() {
-    return (
-      <Page>
-        <HeroUnit image="img/hero-unit.jpg">
-          <h1>Unlock opportunities for all citizens of the Web.</h1>
-          <div><Ia href="/foo/" className="btn btn-awsm">Join Us</Ia></div>
-        </HeroUnit>
-        <Values/>
-        <CaseStudies/>
-      </Page>
-    );
-  }
-});
-
-module.exports = Homepage;
+module.exports = function() {
```

Comment:
So this is definitely the most unintuitive part of this commit.

Due to the way that [React's reconciliation algorithm](http://facebook.github.io/react/docs/reconciliation.html) works, we need to have a `<Page>` component be the top-level component of all our pages if we want to have cool transitions between them. If we don't, React's reconciliation algorithm will think we're rendering completely different pages and rebuild the DOM of the new one from scratch, thus not triggering any CSS transition. Hence we need to get rid of the `<Homepage>` top-level component.

That said, I'm not a big fan of the particular *way* that I got rid of the `Homepage` component. Might be better to have `Homepage` be a component that goes *inside* a page (and is wrapped by a `<div>`); that way it can still be a React component rather than a weird factory function, and we can still have our cool transition effects.